nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -410,11 +410,11 @@ in
       etc."qemu/bridge.conf".text = lib.concatMapStringsSep "\n" (e: "allow ${e}") cfg.allowedBridges;
       systemPackages = with pkgs; [
         libressl.nc
-        iptables
+        config.networking.firewall.package
         cfg.package
         cfg.qemu.package
       ];
-      etc.ethertypes.source = "${pkgs.iptables}/etc/ethertypes";
+      etc.ethertypes.source = "${config.networking.firewall.package}/etc/ethertypes";
```

Comment:
The default value for firewall.package is `if config.networking.nftables.enable then "pkgs.nftables" else "pkgs.iptables"`.

`nftables` doesn't contain `/etc/ethertypes` so this wouldn't work when using nftables, right?

Reply:
Good catch! Let's keep `pkgs.iptables` for this one then.